apps/src/templates/studioHomepages/teacherHomepageV2/sectionAvatars/section-avatars.module.scss

```diff
@@ -0,0 +1,149 @@
+@use '@code-dot-org/component-library-styles/typography.module.scss' as
+  typography;
+@import 'color';
+@import 'https://fonts.googleapis.com/css2?family=Noto+Color+Emoji&display=swap';
+
+// Section Avatars
+.sectionAvatarSmall {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 56px;
+  height: 56px;
+  border-radius: 50%;
+  font-family: 'Noto Color Emoji', sans-serif;
+  font-size: 32px;
+  margin-inline-end: 12px;
+}
+
+.sectionAvatarLarge {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 98px;
+  height: 98px;
+  border-radius: 50%;
+  font-family: 'Noto Color Emoji', sans-serif;
+  font-size: 56px;
+  margin-inline-end: 12px;
+}
+
+.sectionAvatarXL {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 140px;
+  height: 140px;
+  border-radius: 50%;
+  font-family: 'Noto Color Emoji', sans-serif;
+  font-size: 80px;
+  margin-inline-end: 12px;
+}
+
+// Section avatar editor
+.editorDialog {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  min-width: 600px;
+  max-width: 800px;
+  padding: 24px 32px;
+  border-radius: 4px;
+  gap: 16px;
+  background-color: var(--white);
+
+  h3 {
+    margin-bottom: 0.5em;
```

Comment:
Very minor optional nit: would it make sense to be consistent about em vs px in this file?